File: Question2.c

//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//  File Name       : Question_2.c
//  Description     : Convert USD to INR 
//  Author          : lina vijay patil
//  Date            : 22/10/2025
//
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
#include<stdio.h>
int USDtoINR(int iNo)
{
    int iINR = 0;
    iINR = 70 * iNo;
    return iINR;
}

int main()
{
    int iValue = 0 , iRet = 0;

    printf("Enter USD\n");
    scanf("%d",&iValue);

    iRet = USDtoINR(iValue);

    printf("%d Dollars is %d INR",iValue,iRet);

    return 0;
}

File: Question2.java

////////////////////////////////////////////////////////////
//
//  File name :     Question2.java
//  Description :   To find out how many odd and even 
//                  numbers are present between 1 and N
//  Author :        lina patil
//  Date :          20/11/2025
//
////////////////////////////////////////////////////////////

class Logic
{
    void CountEvenOddRange(int iNo)
    {
        int iCnt = 0;
        int iEvenCount = 0;
        int iOddCount = 0;

        for (iCnt = 1; iCnt <= iNo; iCnt++)
        {
            if (iCnt % 2 == 0)
            {
                iEvenCount++;
            }
            else if (iCnt % 2 != 0)
            {
                iOddCount++;
            }
        }

        System.out.println("Count of Even Numbers: " + iEvenCount);
        System.out.println("Count of Odd Numbers: " + iOddCount);
    }
}

class Question2
{
    public static void main(String A[])
    {
        Logic lobj = new Logic();
        int iValue = 50;

        lobj.CountEvenOddRange(iValue);
    }
}
